.gitignore

```diff
@@ -18,3 +18,7 @@
 /plugins/de.ovgu.featureide.examples/featureide_examples/BankAccount-FH-JML/src
 /plugins/de.ovgu.featureide.examples/featureide_examples/ExamDB-FH-JML-withoutPMR/src
 /plugins/de.ovgu.featureide.examples/featureide_examples/StringMatcher-FH-JML/src
+/.metadata/
+/.recommenders/
```

Comment:
What are these two entries used for?

Reply:
Files under this directories are shown as changed, I've moved the entries into the file ".git/info/exclude", this way the local stuff will not show up in the global repository .gitignore